src/style.css

```
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 500;
+  font-style: "Book";
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: "normal";
+}
+
+.container * {
+  border: none;
+  
+}
```

```suggestion

.container * {
  border: none;
}
```

you have container selector below, do you really need this?